### PR TITLE
fix faq

### PR DESCRIPTION
**Pull request by alidevjimmy**
This PR fixes FAQ page bug after #25 commit.

### CODE REVIEW

File: layouts/about/faq.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h1 class="md:text-4xl text-2xl text-bold text-neutral-50 font-medium text-left"
   </div>
   <div class="mt-10 bg-[#141B25] rounded-xl p-10 md:p-20" id="grouds-bundler">
     {{ range $i,$group := .Site.Data.faq }}
-    {{ $group_id := lower $group.title }}
```
Comment (b00f) [inline layouts/about/faq.html:58]: It is better for the ID to be in lowercase. Why did you remove it

Comment (alidevjimmy) [reply layouts/about/faq.html:58]: it caused bug due to mismatch with buttons values.
let me make both of lower.
